crates/sui-framework/packages/sui-system/sources/sui_system_state_inner.move

```diff
@@ -290,9 +432,19 @@ module sui_system::sui_system_state_inner {
     /// At the end of the epoch, the `validator` object will be returned to the sui_address
     /// of the validator.
     public(friend) fun request_remove_validator(
-        self: &mut SuiSystemStateInner,
+        self: &mut SuiSystemStateInnerV2,
         ctx: &mut TxContext,
     ) {
+        // Only check min validator condition if the current number of validators satisfy the constraint.
+        // This is so that if we somehow already are in a state where we have less than min validators, it no longer matters
+        // and is ok to stay so. This is useful for a test setup.
+        if (vector::length(validator_set::active_validators(&self.validators)) >= self.parameters.min_validator_count) {
```

Comment:
Can we add a test where this path is exercised?

Reply:
This is already tested/exercised in sui_system_tests::test_staking_pool_mappings